README.md

```diff
@@ -482,6 +482,63 @@ export { handler as default } from '@/next-video';
 ```
 
 
+## Default Player
+
+The default player is built with [Media Chrome](https://github.com/muxinc/media-chrome).
+
+- The default theme is [Sutro](https://player.style/themes/sutro) by Mux.
+- The video engine changes automatically based on the source format:
+  - Video files (like MP4, MP3, WEBM) that are progressively downloaded are played with the native `<video>` element.
+  - HLS streams are played with [`<mux-video>`](https://github.com/muxinc/elements/tree/main/packages/mux-video).
```

Comment:
Should HLS streams be played with `<hls-video>` and _Mux_ assets played with `<mux-video>`? What about things like DASH?

Reply:
that would make it more 1 to 1 yes. I'll see if I can add in castable-video in both hls-video and dash-video.